playground/index.ts

```diff
@@ -0,0 +1,82 @@
+import { load } from "../src";
+import BotDetector from "../src/detector";
+import "./style.css";
+
+const runDetection = async () => {
+  const statusEl = document.getElementById("status");
+  const resultTableEl = document.getElementById("result-table");
+  const errorSectionEl = document.getElementById("error_section");
+  const resultsSectionEl = document.getElementById("result_section");
+  const errorEl = document.getElementById("error");
+  const perfEl = document.getElementById("perf");
+  const timeEl = document.getElementById("time");
+  const sourcesEl = document.getElementById("sources");
+  const resultEl = document.getElementById("result");
+
+  statusEl!.textContent = "Loading...";
+  resultTableEl!.innerHTML = "";
+  errorSectionEl!.style.display = "none";
+  resultsSectionEl!.style.display = "none";
+
+  try {
+    const t0 = performance.now();
+
+    const instance = await load();
+    const collectTime = performance.now() - t0;
+
+    const result = instance.detect();
+    const processTime = performance.now() - t0;
+
+    errorSectionEl!.style.display = "none";
+    resultsSectionEl!.style.display = "block";
+
+    perfEl!.textContent = collectTime.toFixed(0) + " ms";
+    timeEl!.textContent = processTime.toFixed(0) + " ms";
+
+    const components = (instance as BotDetector)["components"];
```

Comment:
This code uses an undocumented API to demonstrate a basic feature. Maybe this should be a part of the public API, similar to how FingerprintJS returns the components.